core/scripts/upgrade_dbt_schema_tests_v1_to_v2.py

```diff
@@ -395,7 +396,7 @@ class CustomSortedColumnsSchema(CustomSortedSchema):
 
 
 for cls in (CustomSortedRootSchema, CustomSortedModelsSchema, CustomSortedColumnsSchema):
-    yaml.add_representer(cls, cls.representer)
+    yaml.add_representer(cls, cls.representer)  # type: ignore [attr-defined]
```

Comment:
@nathaniel-may You might have a solution to this one.. but for the life of me I couldn't sort out why mypy thinks `cls.representer` was undefined.  It's clearly in all three classes above.

Reply:
So why does it find `ABCMeta` as the common type bound? Shouldn't it be `CustomSortedSchema`?  All items in the tuple are subclasses of it...

Either way, I'll just leave it ass an ignore for now.

Reply:
oops I'm totally wrong here. The tuple is of type `Tuple3[Type[CSRS], Type[CSMS], Type[CSCS]]` since those are all classes not class instances.

I'd expect the following to type check, but it doesn't.
```python
from typing import Iterator, Type, List
boop: Iterator[Type[CustomSortedSchema]] = (CustomSortedRootSchema, CustomSortedModelsSchema, CustomSortedColumnsSchema).__iter__()
```
```
error: Incompatible types in assignment (expression has type "Iterator[ABCMeta]", variable has type "Iterator[Type[CustomSortedSchema]]")
```

I'm actually not sure why it's not unifying those types correctly.

In any case doing it on 3 separate lines will fix it since the problem is definitely with the conversion to an iterator.

Reply:
Tracked it down to the point where it works fine if the parent class doesn't inherit from `dict` it works. I'll probably dig farther than this warrants for my own understanding.

Reply:
Do we even need this script? It's pretty old and we've been on v2 of schema files for years. Maybe it should just be removed?

Reply:
yeah, let's :knife: it

the link to this script is pointing to the long-unused `development` branch anyway: https://discourse.getdbt.com/t/migrating-to-schema-yml-v2/111